### ISSUE TITLE
Can't click flags to see specific versions of pages.

### ISSUE THREAD

**Issue by WardCunningham** [fedwiki/wiki-plugin-activity#21]
We appear to have lost the ability to click site flags to see pages. I installed the npm version to confirm that this isn't some temporary bug due to running developmental code in production.

```
root@asia:~/modules/wiki# npm install wiki-plugin-activity
wiki-plugin-activity@0.2.5 node_modules/wiki-plugin-activity
└── virtual-dom@2.1.1 (browser-split@0.0.1, x-is-array@0.1.0, x-is-string@0.1.0, next-tick@0.2.2, is-object@1.0.1, ev-store@7.0.0, error@4.4.0, global@4.3.0)
```

Here is a link where I use this just installed Activity plugin and can't open pages by clicking links.

http://ward.asia.wiki.org/view/welcome-visitors/view/recent-changes/bigone.wikity.club/dart

When I recall the same pages with wikity.club as origin I find that flag clicking still works. Perhaps there is a way to tell what version of the activity plugin wikity is serving?

http://bigone.wikity.club/view/welcome-visitors/view/dart/ward.asia.wiki.org/recent-changes

**Comment by paul90** [comment]
Fixed in + wiki-plugin-activity@0.2.6

**Comment by WardCunningham** [comment]
Sweet. Now I can get back to studying the goals pod pages.
